mlflow/tracking/_tracking_service/utils.py

```diff
@@ -190,19 +190,28 @@ def _get_databricks_uc_rest_store(store_uri, **_):
     )
 
 
+global _tracking_store_registry
 _tracking_store_registry = TrackingStoreRegistry()
```

Comment:
Do we need this change ? `_tracking_store_registry` is already a global variable.

Reply:
Right. Changed.